packages/serverless-collector/src/index.js

```diff
@@ -21,11 +29,14 @@ const { environment: environmentUtil } = require('@instana/serverless');
 const isExcludedFromInstrumentation = coreUtil.excludedFromInstrumentation && coreUtil.excludedFromInstrumentation();
 
 if (!isExcludedFromInstrumentation) {
+  logger.debug('Validating environmentUtil');
```

Comment:
Do we really need these logs 🤔

Reply:
These are conditionwise logs, maybe we don't need these. Anyway, I am moving these serverless logs to another PR as I focused on agent logs for now.